sqlglot/dialects/clickhouse.py

```diff
@@ -332,6 +332,9 @@ class Parser(parser.Parser):
             TokenType.SET,
         }
 
+        RESERVED_TOKENS = parser.Parser.RESERVED_TOKENS.copy()
+        RESERVED_TOKENS.remove(TokenType.SELECT)
```

Comment:
This is what Toby suggested:
```suggestion
        RESERVED_TOKENS = parser.Parser.RESERVED_TOKENS - {TokenType.SELECT}
```